docs/markdown/pipelines/rnaseq.md

```diff
@@ -31,23 +31,73 @@ Example command (change `cfc` for `binac` on binac cluster):
 
 ```bash
 #!/usr/bin/bash
-nextflow run qbicsoftware/rnadeseq -r 1.3.2 -profile docker \
---rawcounts 'merged_gene_counts.txt' \
---metadata 'QXXXX_sample_preparations.tsv' \
+nextflow run qbic-pipelines/rnadeseq -r 2.4.1 -profile cfc \
+--gene_counts 'merged_gene_counts.txt' \
+--input 'QXXXX_sample_preparations.tsv' \
 --model 'linear_model.txt' \
 --contrast_matrix 'contrasts.tsv' \
 --project_summary 'QXXXX_summary.tsv' \
 --multiqc 'MultiQC.zip' \
---quote 'QXXXX_signed_offer.pdf' \
---versions 'software_versions.csv' \
---report_options 'report_options.yml' \
---species Hsapiens
+--software_versions 'software_versions.csv' \
+--outdir 'results'
+```
+
+### Docker containers
+
+After the release of rnadeseq 2.1, the docker containers were moved from Docker Hub to the GitHub Container Registry (ghcr) as Docker announced a change of free subscriptions. For all rnadeseq versions >2.1, this is reflected in the pipeline, so you don't need to do anything.
+
+For version 2.1 or later, should you have trouble executing the pipeline because the container was removed from Docker Hub (this could for example lead to an error like "docker: Error response from daemon: manifest for qbicpipelines/rnadeseq:2.1 not found: manifest unknown: manifest unknown."), please save the following code to a `container.config` (change 2.1 to the version you want to use):
```

Comment:
you mean "2.1 or previous versions"?

Reply:
Sorry Sabrina, I don't understand you question. If you refer to the sentence `(change 2.1 to the version you want to use)`, that means that in the small config code, the user should replace 2.1 with the desired version number

Reply:
2.1 or earlier!!!